src/js/testem-component.js

```diff
@@ -125,16 +125,10 @@ fluid.testem.getTestemOptions = function (that) {
 
 fluid.testem.generateRimrafWrapper = function (path, rimrafOptions) {
     return function () {
-        var rimrafPromise = fluid.promise();
-        rimraf(path, fluid.copy(rimrafOptions), function (rimrafError) {
-            if (rimrafError) {
-                rimrafPromise.reject(rimrafError);
-            }
-            else {
-                rimrafPromise.resolve();
-            }
-        });
-        return rimrafPromise;
+        var fluidPromise = fluid.promise();
+        var rimrafPromise = rimraf(path, fluid.copy(rimrafOptions));
```

Comment:
Any particular reason we can't just return the upstream promise?

Reply:
I was a bit worried that I might have used an infusion convention somewhere (i.e. calling `reject`), but the tests pass with this streamlined.